flaskbb/templates/forum/forum.html

```diff
@@ -89,6 +91,12 @@
                                 {{ topic.username }}
                                 {% endif %}
                             </div>
+
+                            {% if topic.hidden %}
+                            <div class="topic-author">
+                                {% trans %}Hidden on{% endtrans %} {{ topic.hidden_at|format_date('%b %d %Y') }} {% trans %}by{% endtrans %} {{ topic.hidden_by.username }}
```

Comment:
Is there a better way to do this? I poked through the Jinja2 i18n docs, and they seem to recommend doing like:

```html
{{ gettext("Hidden on %(date)s by %(username)s") | format(date=..., username=...) }}
```

But I don't know if we have prior art in FlaskBB for that. However, it is arguably a better way to do translations since the meaning of the phrase can be translated rather than bits and pieces.

Reply:
IIRC it can be done like this in templates:
```jinja
{% trans username=topic.hidden_by.username date=format_date(topic.hidden_at, '%b %d %Y') %}Hidden on {{ date }} by {{username}}{% endtrans %}
```